src/components/layout/modal/browser_update_alert_modal.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useEffect } from 'react'
+import { ModalRefType } from 'components/elements/modal'
+import { Localize } from 'components/localization'
+import { Modal, Header } from 'components/elements'
+import { useUserBrowser } from 'components/hooks/use-user-browser'
+import { CookieStorage } from 'common/storage'
+
+const BrowserUpdateAlertModal = () => {
+    const modalRef = React.useRef<ModalRefType>()
+    const { is_outdated } = useUserBrowser()
+    const cookie_key = 'browser_update_alert_modal_shown'
```

Comment:
As a suggestion you can move `cookie_key` out of the component so in each render it doesn't create it again!